frontend/scripts/constants.js

```diff
@@ -1,3 +1,4 @@
+export const PROFILE_PAGES_WHITELIST = ['EXPORTER', 'IMPORTER', 'MUNICIPALITY'];
```

Comment:
I still believe we can get rid of this. Nodes have their own profile type, which is only set in nodes of these 3 types.